### ISSUE TITLE
Connection ( red/green light) doesn't work

### ISSUE THREAD

**Issue by Ontrumat** [oxen-io/session-desktop#2836]
Hi,
so far very happy with Session !
But, today I got a notification that there was an update for Session, so I did.But  after installation, I got a text box asking if I wanted to reopen (or  not) Session and after acknowledging it, I got a black screen. 
Nothing  came up. Tried re-install but nothing.
So, after a while I looked for  older versions of Session and installed them ( the one mentioned  earlier ( 1.7.7 & 1.8.6 ) and they both worked up till the point  that they don't connect to the net. 
The red light button that suppost to  get green ones Session is connected to the net just doesn't want to do  the job... tried re-install but nothing worked.
So, here I am hoping someone could help me out with this on...
Oh, I'm using an older version of Mac plarform( High Sierra).Thank you for your time and help.
Kind regards,
-- | --

![Screen Shot 2023-07-20 at 15 34 52](https://github.com/oxen-io/session-desktop/assets/101600192/1a0645ce-4079-4863-9f01-e3ff1a2b05c3)

**Comment by Ontrumat** [comment]
I got in touch with Session support and they advices me to do this. ( which didn't work btw, but if it happens to anybody out there it might work...)

First, I suggest we start Session from scratch:
1. Open the Finder.
2. Press Cmd+Shift+G.
3. Enter this in the "Go to" address bar: ~/Library/Application Support
4. Locate the Session folder and rename it to something like Session-old for backup purposes.

After you've done this, open Session again. You can either link your device or restore your account using your recovery phrase.

**Comment by Ontrumat** [comment]
Ok, I found a work around...

Since I had a Seesion version on my laptop, I copied the Session folder in the application folder (Enter this in the "Go to" address bar: ~/Library/Application Support) from the laptop and got rid of the Session folder situated in the Application Folder on my desktop. Then I copy/pasted the Session version of the laptop into the application folder from my desktop Mac.

I Looked for the Session version (https://session.en.uptodown.com/mac/versions) that finally "adapted" the folder and made it all work again.

Done.

Hope this helps someone out there in the future.
